### ISSUE TITLE
Quick test commands

### ISSUE THREAD

**Issue by EvanSkiStudios** [Mercerenies/a-turtles-troll#71]
something to trigger an anvil, sand, or stalagtite to fall on your head

something to trigger the pufferfish to rain from the sky

a command to summon an angle, cake or chest in front of you

mainly quick things i can do to test certain things

**Comment by Mercerenies** [comment]
Yes, definitely need to do this. Thanks for the nudge; I've been meaning to.

**Comment by Mercerenies** [comment]
In no particular order, features that I think could benefit from a command.
* [ ] Lava launch effect
* [ ] Fall damage effect
* [ ] Axolotl death simulator
* [ ] Angel summon
* [ ] Mimic summon
* [ ] New pet phantom (vector) summon
* [ ] Armored wither skele summon
* [ ] Ender crystal summon
* [ ] Pufferfish rain
* [ ] AK47 pillager summon
* [ ] Command to give out free cookies (as though it's dawn/dusk)
* [ ] Anvil drop command
* [ ] Explosive arrow "give" command
* [ ] Command to summon a gravestone at the player's current position
* [ ] Command to automatically appease the gods
* [ ] Command to get a new request from the gods
* [ ] Command to kill self of old age

**Comment by Mercerenies** [comment]
Added lavalaunch command: f0be4c1

**Comment by Mercerenies** [comment]
Added mimix command: 300d31b

**Comment by Mercerenies** [comment]
Added pufferfish command (as part of migration to #14): 512482d
